ecosystem/platform/server/app/views/settings/connections.html.erb

```diff
@@ -1,32 +1,34 @@
 <% content_for(:page_title, 'Connection Settings') %>
 <h3 class="text-4xl mb-8 font-display font-light">Connections</h3>
 <% User.omniauth_providers.each do |provider| %>
-  <% authorization = @authorizations[provider.to_s] %>
-  <div class="mb-8 flex justify-between">
-    <div class="flex gap-4 items-center">
-      <%= render IconComponent.new(provider, size: :large) %>
-      <div>
-        <h4 class="text-xl font-display font-light"><%= provider.to_s.titleize %></h4>
-        <% if authorization %>
-          <p><%= authorization.display_shortname %></p>
-        <% end %>
+  <% authorizations = (@authorizations[provider.to_s] || []) + [nil] %>
```

Comment:
Any way to avoid doing `+ [nil]`?

Reply:
not that I know of without duplicating some code